COMETwebapp/COMETwebapp/Program.cs

```diff
@@ -22,14 +22,27 @@
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
+using Blazored.Modal;
+using CDP4Dal;
 using COMETwebapp;
+using COMETwebapp.SessionManagement;
+using Microsoft.AspNetCore.Components.Authorization;
 using Microsoft.AspNetCore.Components.Web;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 
 var builder = WebAssemblyHostBuilder.CreateDefault(args);
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
-builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddBlazoredModal();
```

Comment:
I assume this is the whole new c# method of not explicitly wrapping into namespaces, classes and methods. I dont know @samatrhea whether we should do this yet.

Reply:
we can leave it like this for now, if we see that the CI/CD does not work we can make it a full/explicit program class that might be easier to read for all developers not yet familiar with the new style